payment-processor/src/main/kotlin/org/ostelco/prime/paymentprocessor/PaymentProcessorModule.kt

```diff
@@ -40,8 +50,35 @@ class PaymentProcessorModule : PrimeModule {
         env.lifecycle().manage(StripeEventPublisher)
         env.lifecycle().manage(StoreStripeEvent())
         env.lifecycle().manage(ReportStripeEvent())
-        env.lifecycle().manage(RecurringPaymentStripeEvent())
     }
+
```

Comment:
You will have to add a cleanup function for local testing. See the `scaninfo-datastore` module

```
    fun cleanup() {
        if (ConfigRegistry.config.storeType == "inmemory-emulator") {
            // Stop the emulator after unit tests.
            localDatastoreHelper.stop()
        }
    }
```

Reply:
Will add unit tests along the lines of what is done in `scaninfo-datastore` before merge.